apps/laboratory/src/utils/EIP5792Utils.ts

```diff
@@ -19,7 +26,11 @@ export function getAtomicBatchSupportedChainInfo(
     const walletCapabilities = walletCapabilitiesString && parseJSON(walletCapabilitiesString)
```

Comment:
I'd move the capabilities calculation to the upper context and keep the same interface for viem and ethers. wyt?
would that remove the need to duplicate the function? (even if not, I think it's worth)

Reply:
As per ERC the capability type is defined as
>type GetCapabilitiesResult = Record<`0x${string}`, <Record<string, any>>; // Hex chain id

but viem is returing the chainId as number i.e 
>type GetCapabilitiesResult = Record<number, <Record<string, any>>;

That is why, I tried ethers implementation as per ERC and didn't create the same interface for viem and ethers, I can process the data  and then create the common interface though.

Reply:
would make sense but don't think it's mandatory. would make the code nicer tho. Will drop an approve, if you agree  the abstraction makes sense let's do it else just go ahead and merge 🖖